src/Mod/Fem/femresult/resulttools.py

```diff
@@ -68,6 +68,16 @@ def purge_results(analysis):
             analysis.Document.removeObject(m.Name)
     analysis.Document.recompute()
```

Comment:
This should also be modified to remove reports from other solvers besides Calculix

Reply:
Ok, done. Only CalculiX and Elmer create such text documents.

Reply:
It is preferable not to use the object names here. If the name is changed or reports from other solvers are added, the purge will fail.
For the moment, it is better to remove all `App::TextDocument` objects.

Reply:
Ok, PR updated.